my.css

```diff
@@ -352,3 +352,162 @@ body {
     justify-content: center;
   }
 }
+
+/* Вторая секция */
+.bagscluth__content {
+  display: flex;
+  
+}
+.bagscluth {
+  margin: 0;
+  padding: 0;
+  position: relative;
+  max-height: 900px;
+  display: flex;
+  background-color: #fae8e4;
+}
+.bagscluth__img {
+  width: 55%;
+  height: 844px;
+  
+  display: flex;
+  align-items: center;
+}
+.bags-img {
+  position: absolute;
+  display: block;
+  width: 340px;
+  height: 370px;
+  padding-left: 90px;
+
+}
+.bags-img1 {
+  position: absolute;
+  width: 540px;
+  height: 740px;
+  left: 160px;
+}
+
+.inner {
+  width: 400px;
+  position: relative;
+  right: 10px;
+  padding: 50px;
+
+}
+.card {
+  
+  width: 350px;
+  height: 456px;
+  padding: 40px 55px 45px 65px; 
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected newline after ";" in a multi-line declaration block (declaration-block-semicolon-newline-after)](https://app.codacy.com/gh/WebHeroSchool/itog/pullRequest?prid=5642160)